packages/FormElement/stories/examples/ButtonGroupExample.js

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import { Tagline } from "storybook/assets/styles/common.styles";
+import ButtonGroup from "@paprika/button-group";
+import FormElement from "../../src/FormElement";
+
+export default function ButtonGroupExample() {
+  const refButtonGroup = React.useRef(null);
+  const buttonOptionsArray = ["Black Panther", "Wonder Woman", "Spiderman", "The Incredibles", "Thor"];
+  const buttonGroupOptions = buttonOptionsArray.map(hero => (
+    <ButtonGroup.Item value={hero} key={hero}>
+      {hero}
+    </ButtonGroup.Item>
+  ));
+
+  return (
+    <>
+      <Tagline>Form Element with ButtonGroup.</Tagline>
```

Comment:
These `<Taglines>` can be moved to a `tagline` prop for the `<ExampleStory>` helper.  For example, this one can go on this line instead:
https://github.com/acl-services/paprika/pull/741/files#diff-3e0c334c0f1c2f089735a88e0dc4ed84R59